### DISCUSSION TITLE
Add button/URL to return directly to the Specimen Uploader following data ingestion

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#416] [Wish List]
**Requested by collection administrator:**
> Having an “upload again” or “continue uploading” button after you upload one text file might be nice, rather than having to navigate back to the upload page.

**Related screenshot:**
- Current view users see following data ingestion:
![Screenshot 2023-11-16 at 2 57 37 PM](https://github.com/BioKIC/symbiota-docs/assets/11181096/5ba16271-837b-46cc-9347-6cbeb65d8899)

**Comment by themerekat** [comment]
@ljwalker , if you click the "List of Upload Profiles" breadcrumb, it will take you back to the upload profiles, where you can start the uploading process again. Do you think that's what the user is looking for?
